docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -207,7 +207,13 @@ async fn main() -> Result<()> {
 
 ## Filtering event queries
 
-To filter the events returned from your queries, use the following data structures. Currently, you cannot combine filters.
+To filter the events returned from your queries, use the following data structures.
+
+:::caution
```

Comment:
The use of alerts and emphasis may be overkill here -- please steer me in the right direction @ronny-mysten !

Reply:
Is a caution warranted? I'd suggest 'info' unless there is a chance of bodily harm ;) or a bug that might go unnoticed if this info is missed. It's a nit, though, as the callout structure is not as important as the information it contains.

Reply:
It's a fair question -- I wanted to draw the eye to this section because people kept missing the detail that there were two different kinds of filter, but I think `info` would be fine too!